utils/harp.dependencies.js

```diff
@@ -1,34 +1,13 @@
-const cpx = require('cpx'),
-  chalk = require('chalk');
-
+const chalk = require('chalk');
+const folderCopy = require('./file-copy');
 const message = chalk.red.bold('Be careful moving around the folder structure, make sure all the necessary files are always copied');
 console.log(message);
 
-
-const files = {
+folderCopy({
   'src/main.js': 'www',
   'src/tools/prism.js': 'www/tools',
   'src/vendors/material-design-lite/material.min.js': 'www/vendors/material-design-lite',
   'src/vendors/material-design-lite/material.min.css': 'www/vendors/material-design-lite',
   'src/images/**/*': 'www/images',
-  'src/pages/docs/img/*': 'www/pages/docs/img',
```

Comment:
Oh yes I also moved the image files under the guide folder! Check and image is getting loaded correct in the architecture page.